src/sinks/loki/sink.rs

```diff
@@ -379,7 +396,7 @@ impl LokiSink {
                     None
                 }
             })
-            .request_builder(NonZeroUsize::new(1), self.request_builder)
+            .request_builder(NonZeroUsize::new(50), self.request_builder)
```

Comment:
I'm not sure why this was pinned to 1. It seems like the concurrency limit at the top should stop this layer from having more than one request in flight.

Reply:
The concurrency limit was on the inner service itself, which comes after request builder.  That said, however: the request builder is concurrent but _also_ ordered.

Reply:
Aha, I see. Then I suppose I do probably want to keep this conditional based on the configured `out_of_order` action then. I think for the requests that require timestamp rewriting, they do need to be processed in order.